src/components/Main.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const Main = props => (
+  <section className="main">
+    <input id="toggle-all" className="toggle-all" type="checkbox" />
```

Comment:
Input should be checked when `item.done` is true.

Reply:
![image](https://user-images.githubusercontent.com/29413892/59746826-59b4c880-9280-11e9-8d7f-14bb8e9f135a.png)